### ISSUE TITLE
primer list not found

### ISSUE THREAD

**Issue by 4938394094** [artic-network/artic-ncov2019#43]
The link to get the primer list for sars-CoV-2 does not work
https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019/V3

**Comment by will-rowe** [comment]
Hi.

I'm afraid that link has broken since I updated the primer scheme repository. I thought I had updated all of the links. Can you let me know where this link is found and I can update it to the latest (which is: https://github.com/artic-network/primer-schemes/tree/master/nCoV-2019/V3 )

**Comment by 4938394094** [comment]
Thank you!
On this website, there are two places:

https://artic.network/ncov-2019
1- Updates   
   - 24-Mar-2020: We have posted an update to the amplicon set (V3) (in the document the link is broken)   



2- Resources and documents

   
   - nCoV-2019 PrimalSeq sequencing primers

link: Github (version 3) (also on the same page this link is broken)








    Le mercredi 16 septembre 2020 11 h 44 min 42 s HAE, Will Rowe <notifications@github.com> a écrit :  
 
 


Hi.

I'm afraid that link has broken since I updated the primer scheme repository. I thought I had updated all of the links. Can you let me know where this link is found and I can update it to the latest (which is: https://github.com/artic-network/primer-schemes/tree/master/nCoV-2019/V3 )

**Comment by will-rowe** [comment]
Thanks for flagging these. I've just added a pull request to fix them.

Background: we recently swapped to a submodule to make primer schemes easier to maintain, test and version. The downside was that old links break as they can't follow the submodule. The new pull request swaps to a git subtree instead, which should fix the broken links.

**Comment by 4938394094** [comment]
Thank you.  I'm just happy to get the primer sequences for free because I only need to sequence a part of the genome, and IDT is selling only the whole primer set.

    Le jeudi 17 septembre 2020 07 h 00 min 25 s HAE, Will Rowe <notifications@github.com> a écrit :  
 
 


Thanks for flagging these. I've just added a pull request to fix them.

Background: we recently swapped to a submodule to make primer schemes easier to maintain, test and version. The downside was that old links break as they can't follow the submodule. The new pull request swaps to a git subtree instead, which should fix the broken links.
